contracts/common/Vesting/Vesting.sol

```diff
@@ -35,6 +36,17 @@ abstract contract Vesting is APRCalculatorConnector {
     // solhint-disable-next-line func-name-mixedcase
     function __Vesting_init_unchainded() internal onlyInitializing {
         vestingLiquidityDecreasePerWeek = 133; // 0.0133
+        penaltyDecreasePerWeek = 50; // 0.5%
+    }
+
+    // _______________ External functions _______________
+
+    /**
+     * @inheritdoc IVesting
+     */
+    function setPenaltyDecreasePerWeek(uint256 newRate) external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
Let's create a reusable modifier to wrap ```onlyRole(DEFAULT_ADMIN_ROLE)```  - ```onlyGovernance()```` so we can use it everywhere for better readability

Reply:
I agree, wouldn't that be better to be in the ticket (governance consistency) so I can modify it in every contract?